locale/templates/auth.ftl

```diff
@@ -20,6 +20,9 @@
 # "Firefox Cloud" should be treated as a brand.
 -product-firefox-cloud = Firefox Cloud
 
+# Other brands
+-paypal = PayPal
```

Comment:
should consider change brand names to `-brand-paypal` to differentiate brand/product names from others.